### ISSUE TITLE
Reset all answers button

### ISSUE THREAD

**Issue by dattvidc** [nas5w/typeofnan-javascript-quizzes#9]
I'd like to make a "Reset all answers" button on the main page. 
the button when clicked would re-set all answers back to nothing.

**Comment by dattvidc** [comment]
Also...let me know if I am doing this wrong. I read the docs that indicated an issue would be the right place to start. This would be my first time pushing code on Github, so I understand if you would rather not.

**Comment by nas5w** [comment]
Love the idea! Go forth and conquer, let me know if you need any help!

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by nas5w** [comment]
Also, I’m ecstatic this is your first github contribution! That makes me really happy.

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by nas5w** [comment]
@dattvidc Do you think you'll get around to this? Would be really happy if you could!

**Comment by dattvidc** [comment]
Yes. I keep odd hours but I am back online Tuesday, and will start working on it there

**Comment by nas5w** [comment]
Awesome, thanks!

**Comment by dattvidc** [comment]
This might take a little while longer. Ran into a problem installing. NodeJS version on ubuntu was not compatible with YARN and surprisingly updating node.js on ubuntu requires several other utilities. 
I am on it though...Not MIA

**Comment by dattvidc** [comment]
Pull Request "Delete all answers" button to frontpage #78
